packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -860,7 +876,11 @@ export function wrapAngularDevkitSchematic(
   // were written with Nx in mind, and may care about tags.
   require('./compat');
 
-  return async (host: Tree, generatorOptions: { [k: string]: any }) => {
+  return async (
+    host: Tree,
+    projectGraph: ProjectGraph,
```

Comment:
We can't add this here, as this is public API surface. We may be able to call createProjectGraphAsync inside the body of the returned function, would that help with the issue you are seeing?